### PR TITLE
Update op-geth dependency to 1.14.8 base

### PR DESCRIPTION
**Pull request by sebastianst**
**Description**

Updates op-geth to https://github.com/ethereum-optimism/op-geth/pull/363

- Fixes https://github.com/ethereum-optimism/client-pod/issues/971

### CODE REVIEW

File: op-program/client/mpt/db.go [hunk 1]
```diff
@@ -51,10 +51,6 @@ func (p DB) Compact(start []byte, limit []byte) error {
 	return nil // no-op
 }
 
-func (p DB) NewSnapshot() (ethdb.Snapshot, error) {
-	panic("not supported")
-}
-
```
Comment (axelKingsley) [inline op-program/client/mpt/db.go:57]: Why this change? (and the other removal of unimplemented function `NewSnapshot` in l2 db)

**Review by axelKingsley — APPROVED** [review]
Approved based on the results of the sync test and the limited (nil) additional changes required to support `1.14.8`

File: op-program/client/mpt/db.go (hunk 1 above)
Comment (protolambda) [reply op-program/client/mpt/db.go:57]: The `NewSnapshot` method was removed from the DB interface in upstream geth (for each DB type implementation). It's not used in the fault-proof, so we can drop it also.

**Review by protolambda — APPROVED** [review]
